lasagne/layers/pool.py

```diff
@@ -221,10 +232,53 @@ def get_output_for(self, input, **kwargs):
                                         st=self.stride,
                                         ignore_border=self.ignore_border,
                                         padding=self.pad,
+                                        mode=self.mode,
                                         )
         return pooled
 
 
+class MaxPool2DLayer(Pool2DLayer):
+    """
+    2D max-pooling layer
+    Performs 2D max-pooling over the two trailing axes of a 4D input tensor.
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or tuple
+        The layer feeding into this layer, or the expected input shape.
+    pool_size : integer or iterable
+        The length of the pooling region in each dimension.  If an integer, it
+        is promoted to a square pooling region. If an iterable, it should have
+        two elements.
+    stride : integer, iterable or ``None``
+        The strides between sucessive pooling regions in each dimension.
+        If ``None`` then ``stride = pool_size``.
+    pad : integer or iterable
+        Number of elements to be added on each side of the input
+        in each dimension. Each value must be less than
+        the corresponding stride.
+    ignore_border : bool
+        If ``True``, partial pooling regions will be ignored.
+        Must be ``True`` if ``pad != (0, 0)``.
+    **kwargs
+        Any additional keyword arguments are passed to the :class:`Layer`
+        superclass.
+    Notes
+    -----
+    The value used to pad the input is chosen to be less than
+    the minimum of the input, so that the output of each pooling region
+    always corresponds to some element in the unpadded input region.
+    """
```

Comment:
For some reason, all empty lines got missing here. Could you please fix this? I guess it won't work in Sphinx this way. (Just copy/paste it again from `Pool2DLayer`, change the first sentence and remove `mode` and `See also`.)